docker/README.md

```diff
@@ -7,7 +7,7 @@ To deploy the Alert Logic Agent Container for Docker, you need your unique regis
 2. Click "Details."
 3. Copy your unique registration key.
 
-## Deploy the Agent Container
+## Deploy the Agent Container Using Standard Docker "Run" Command
```

Comment:
"Using the Standard" ?

Reply:
yes, the standard docker run command.  When you pull swarm into the discussion there are different ways to run containers and swarm doesn't do "run".  So we'll need to separate the difference between running base docker and how it has to be done on swarm